R/method-differentialAbundance.R

```diff
@@ -95,7 +97,10 @@ setGeneric("toJSON",
 setMethod("toJSON", signature("DifferentialAbundanceResult"), function(object, ...) {
   tmp <- character()
 
-  tmp <- paste0('"effectSizeLabel": ', jsonlite::toJSON(jsonlite::unbox(object@effectSizeLabel)), ',')
+  tmp <- paste0(tmp, '"effectSizeLabel": ', jsonlite::toJSON(jsonlite::unbox(object@effectSizeLabel)), ',')
+  tmp <- paste0(tmp, '"pValueFloor": ', jsonlite::toJSON(jsonlite::unbox(object@pValueFloor)), ',')
```

Comment:
if someone sets the pValueFloor to 0, what happens? i guess it just works, returns a floor of 0 here, not sure what for the padjust tbh, and then goes about its day never having realized it didnt actually floor anything? thats maybe ok.. i might feel better if we make a test for it, if we dont have one already.